### PR TITLE
[TASK] Add compiling for sources

### PR DESCRIPTION
**Pull request by benjaminkott**

### CODE REVIEW

File: .ddev/nginx_full/nginx-site.conf
Comment (garvinhicking) [inline .ddev/nginx_full/nginx-site.conf:1]: I think this conflicts with the change to the docroot here: https://github.com/TYPO3-Documentation/render-guides/pull/127#pullrequestreview-1746517155 ?

File: .gitignore
Comment (garvinhicking) [inline .gitignore:1]: I think we should keep the ignore on this temporary directory, this is created by the default documentation export

File: README.rst
Comment (garvinhicking) [inline README.rst:1]: Could you move these instructions to the `Documentation/Developer/Theme-customization.rst` file for example?

File: packages/typo3-guides-extension/src/Command/RunDecorator.php
Comment (garvinhicking) [inline packages/typo3-guides-extension/src/Command/RunDecorator.php:1]: See note above, if we change the DocumentRoot, possibly other parts of the github actions and documentation would need adaptation maybe?

**Review by garvinhicking — COMMENTED** [review]
I can't comment on the changes to the theme itself, just some notes about the surroundings. Thanks for getting involved :-)

File: .ddev/nginx_full/nginx-site.conf
Comment (brotkrueml) [reply .ddev/nginx_full/nginx-site.conf:1]: I will take care of it.

Comment (brotkrueml) [reply .ddev/nginx_full/nginx-site.conf:1]: Done

File: .gitignore
Comment (brotkrueml) [reply .gitignore:1]: I agree.

Comment (brotkrueml) [reply .gitignore:1]: Done

File: README.rst
Comment (brotkrueml) [reply README.rst:1]: Done

File: packages/typo3-guides-extension/src/Command/RunDecorator.php
Comment (brotkrueml) [reply packages/typo3-guides-extension/src/Command/RunDecorator.php:1]: I agree, done. We should always render the docs in the well-known folder.

**Review by brotkrueml — APPROVED** [review]

**Comment by brotkrueml** [comment]
@benjaminkott Thanks for your work on this topic. Very appreciated :+1:

**Pull request merged**